src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updateSubmission/DueDiligence.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updateSubmission;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.AddressDto;
+
+public class DueDiligence {
```

Comment:
There are two types of due diligence. We need to support both (by agent; by overseas entity). Think some more fields are needed.

Reply:
I think this is supported by: https://companieshouse.atlassian.net/wiki/spaces/ROE2/pages/4144005155/ODP-ROE-Update-012-B+Change+list+Calculation#Due-Dilligence